ted_sws/core/model/metadata.py

```diff
@@ -79,9 +79,10 @@ class NormalisedMetadata(Metadata):
     legal_basis_directive: str
     form_number: str
     eforms_subtype: str
-    xsd_version: str
+    xsd_version: Optional[str]
     published_in_cellar_counter: int = Field(default=0)
     is_eform: Optional[bool] = False
```

Comment:
We need to change this flag to a field:
` notice_source: Optional[NoticeSource] = NoticeSource.STANDART_FORMS`
and for NoticeSource as example:
`class NoticeSource(str, Enum):
     STANDART_FORMS = "standart_forms"
     ELECTRONIC_FORMS = "eforms"
     
    def __str__(self):
        return self.value

`